### ISSUE TITLE
Questions for CMUNeXt Training

### ISSUE THREAD

**Issue by WFLiu0327** [FengheTan9/Medical-Image-Segmentation-Benchmarks#2]
Hello author, your article CMUNeXt is great. We have some questions about training with this repository and would like to get your answer:
1. we divide the dataset and train CMUNeXt according to the readme document, however the best results obtained are much lower than the results in the paper as follows:
epoch [212/300]  train_loss : 0.1227, train_iou: 0.8474 - val_loss 0.2993 - val_iou 0.6664 - val_SE 0.7741 - val_PC 0.8161 - val_F1 0.7900 - val_ACC 0.9667 
We believe this may be a problem due to inconsistent dataset partitioning. So if you can could you send me a list of the training set and test set if possible, thank you very much! Email: liuweifeng0327@gmail.com!!!

**Comment by FengheTan9** [comment]
Thanks for using our code.
Split file on BUSI I have sent to your email.

**Comment by sainny** [comment]
I also meetthe same problem, can you share the data set  if possible,Email: liurunhuasainy@gmail.com
